content/_data/changelogs/lts.yml

```diff
@@ -3921,7 +3921,7 @@
   changes: # compared to lts_baseline 2.249 - extracted from the RC commit(s)
 
   - type: security
-    message: Important security fixes from Jenkins 2.252.
+    message: Important security fixes from Jenkins 2.252 and 2.235.4.
```

Comment:
> Security fixes came from the previously released LTS,
> not just from the previously released weekly.

The idea here is to ensure nobody on 2.235.x will think this is new to them (unless on 2.235.3 or older). This isn't "technically correct" but might be helpful.